lib/src/repo_path.rs

```diff
@@ -503,6 +503,17 @@ impl PartialOrd for RepoPathBuf {
     }
 }
 
+impl<P: AsRef<RepoPathComponent>> Extend<P> for RepoPathBuf {
+    fn extend<T: IntoIterator<Item = P>>(&mut self, iter: T) {
+        for pc in iter {
```

Comment:
nit: We generally avoid non-standard abbreviations and acronyms. `component` seems good to me

(sure, this is a very standard acronym but it doesn't mean "personal computer" here :))